include/avif/avif.h

```diff
@@ -305,8 +305,13 @@ typedef enum avifChromaSamplePosition
 
 typedef enum avifRange
 {
-    AVIF_RANGE_LIMITED = 0,
-    AVIF_RANGE_FULL = 1
+    // AVIF_RANGE_LIMITED is only applicable to YUV planes. RGB and alpha planes are always full range.
```

Comment:
Nit: The first word of this comment should be "avifRange" (i.e., the enum type), not "AVIF_RANGE_LIMITED". It means the issue of limited vs. full range is only applicable to YUV planes.

Reply:
This comment is intended to explain why the `avifRange` type is only used for the `yuvRange` member of the `avifImage` struct and is not used with the `avifRGBImage` struct or with the alpha plane in the `avifImage` struct. That's why it is clearer to use "avifRange" in this comment.